context/AuthContext.js

```diff
@@ -10,6 +10,24 @@ export function AuthProvider(props) {
   const [isMentorLoggedIn, setIsMentorLoggedIn] = useState(null);
   const [isUserLoggedIn, setIsUserLoggedIn] = useState(null);
 
+  useEffect(() => {
+    // fetching user from local storage
```

Comment:
We have to decrypt first...see userprofile.js

Reply:
But why, I am not using any value of objects, I am just checking whether userData or mentorData object is there or not